### ISSUE TITLE
"highlight plaintext" -> "language-plaintext"

### ISSUE THREAD

**Issue by Arcovion** [rouge-ruby/rouge#145]
[I read this recently](http://www.w3.org/TR/2011/WD-html5-author-20110809/the-code-element.html) which proposes:

> [...] so that syntax highlighting scripts can use the right rules, may do so by adding a class prefixed with "language-" to the element.

I'm thinking about adding this to `middleman-syntax`, by checking whether `:css_class` ends with a dash, if so then it would be concatenated to the `lexer.tag` instead of being joined by a space.

Also, [`middleman-syntax` interpolates the chosen `:css_class` instead of just using `highlight`](https://github.com/Arcovion/middleman-syntax/blob/arcovion/lib/middleman-syntax/extension.rb#L81). Is this possible within Rouge itself?

Just checking If Rouge can do these things by itself, if not then `middleman-syntax` can easily handle this anyway.

TL;DR: In `rouge/plugins/redcarpet` use the actual `:css_class` [instead of the hard-coded `highlight` class](https://github.com/jneen/rouge/blob/master/lib/rouge/plugins/redcarpet.rb#L19) and concatenate them if the `:css_class` ends with a dash. Also possibly change the default class from `highlight` to `language-`.

Sidenote: Is [this code](https://github.com/jneen/rouge/commit/86e77733695e1f9e3dc12954075c479f59b33469) really necessary? Wondering if I need to bring that hack into `middleman-syntax` or not.
